admin/pkg/docker/docker.go

```diff
@@ -216,23 +216,30 @@ func (b *Base) RunContainer(ctx context.Context, config *container.Config, hostC
 var ErrVolumeExists = fmt.Errorf("VolumeExists")
 
 // CreateVolume first checks if the volume exists and creates it.
-func (b *Base) CreateVolume(ctx context.Context, volumeName string) (*types.Volume, error) {
+func (b *Base) CreateVolume(ctx context.Context, volumeName string, labels map[string]string) (*types.Volume, error) {
 	// We need to first manually check if the volume exists because
 	// cli.VolumeCreate() does not complain if it already exists.
 	v, err := b.Cli.VolumeList(ctx, filters.NewArgs(filters.Arg("name", volumeName)))
 	if err != nil {
 		return nil, err
 	}
 
-	if len(v.Volumes) != 0 {
-		return nil, fmt.Errorf("%w: docker volume with name %q already exists", ErrVolumeExists, volumeName)
+	for _, vol := range v.Volumes {
+		if vol.Name == volumeName {
+			return nil, fmt.Errorf("%w: docker volume with name %q already exists", ErrVolumeExists, volumeName)
+		}
+	}
+
+	volumeLabels := make(map[string]string, 1+len(labels))
+	for k, v := range labels {
+		volumeLabels[k] = v
 	}
 
+	volumeLabels["percona.pmm"] = "server"
```

Comment:
```suggestion
	volumeLabels["percona.pmm.component"] = "server"
```
minor nit: we have labels in the form `percona.pmm.*` (e.g `percona.pmm.created`.maybe we can extend this to follow the same format?

Reply:
Yes, that's a good catch. Will process it in a separate PR.